src/main/resources/swagger/schemas/logged-request.yaml

```diff
@@ -7,22 +7,27 @@ properties:
   url:
     description: The path and query to match exactly against
     type: string
-    example: "/received-request/2"
+    example: "/received-request/2?foo=bar"
   absoluteUrl:
     description: The full URL to match against
     type: string
-    example: "http://localhost:56738/received-request/2"
+    example: "http://localhost:56738/received-request/2?foo=bar"
```

Comment:
If we've got a query parameter here and on line 10 then it also needs to be under the `queryParams` map.